examples/basics/UI.py

```diff
@@ -0,0 +1,217 @@
+import os.path
+
+import gradio as gr
```

Comment:
Since gradio is not added as a dependency, it might be helpful to add something like the following:

```suggestion
try:
    import gradio as gr
except ModuleNotFoundError:
    print(
        "Unable to import the gradio module. Try `pip install gradio`."
    )
    sys.exit(1)
```